Dockerfile

```
@@ -11,4 +11,5 @@ ARG CONDA_PREFIX=/opt/conda
 COPY example.tex /root/example.tex
 
 RUN ${CONDA_PREFIX}/bin/conda install -c conda-forge tectonic texlive-core \
-    && ${CONDA_PREFIX}/bin/tectonic /root/example.tex
+    && ${CONDA_PREFIX}/bin/tectonic /root/example.tex \
+    && curl -o /root/git-latexdiff https://gitlab.com/git-latexdiff/git-latexdiff/-/raw/master/git-latexdiff
```

I would install a specific version, for example the latest tag:
```suggestion
    && curl -o /root/git-latexdiff https://gitlab.com/git-latexdiff/git-latexdiff/-/raw/1.6.0/git-latexdiff
```
If you need a newer version then you can get the raw of specific commit.

---

Yeah, I was not sure about pinning the version. The versions of the conda packages are not pinned either, but this does feel different. (But maybe the conda packages should be pinned too.)